packages/lab/src/calendar/internal/CalendarDay.css

```diff
@@ -44,8 +44,21 @@
 .saltCalendarDay-hoveredOffset:hover {
   background: var(--salt-selectable-background-hover);
 }
+
+.saltCalendarDay-selected {
+  border-radius: var(--salt-palette-corner-weak, 0);
+}
+
 .saltCalendarDay-selectedStart {
   z-index: var(--salt-zIndex-default);
+
+  border-top-left-radius: var(--salt-palette-corner-weak, 0);
+  border-bottom-left-radius: var(--salt-palette-corner-weak, 0);
+}
+
+.saltCalendarDay-selectedEnd {
+  border-top-right-radius: var(--salt-palette-corner-weak, 0);
+  border-bottom-right-radius: var(--salt-palette-corner-weak, 0);
 }
```

Comment:
Is this separate due to specificity issues?

Reply:
❔ do you mean start v.s. end? left vs right ?

Reply:
sorry, I'll be less ambiguous :P `.saltCalendarDay-selected` already exists but this isn't with the previous styles